<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniSyncWindow</class>
 <widget class="QWidget" name="UniSyncWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UniSync</string>
  </property>
  <property name="windowIcon">
   <iconset resource="unisync.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0,0,1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>UniSync</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="paramBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QComboBox" name="cFunc">
        <item>
         <property name="text">
          <string>Select one!</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CREATE - Generate a catalog file from a directory</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DIFF - Compares two directories and print the differences</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CATDIFF - Compares a directory to a catalog file and print the differences</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SYNC - Syncronize a directory to a directory</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAKEUPDATE - Create an update package to sync offline directories</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAKESYNCUPDATE - Create an update package to sync online directories</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>APPLYUPDATE - Apply update package to sync directories</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QPushButton" name="pCatalog">
            <property name="text">
             <string>Catalog file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lCatalog">
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>3</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pSource">
            <property name="text">
             <string>Source folder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lSource">
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>3</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pDestination">
            <property name="text">
             <string>Destination folder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lDestination">
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>3</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pUpdate">
            <property name="text">
             <string>Update folder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lUpdate">
            <property name="text">
             <string>Nonel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Output</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>3</number>
               </property>
               <item>
                <widget class="QRadioButton" name="rVerb0">
                 <property name="text">
                  <string>Quiet</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rVerb1">
                 <property name="text">
                  <string>Verbose</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rVerb2">
                 <property name="text">
                  <string>More verbose</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>File content hashes</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>3</number>
               </property>
               <item>
                <widget class="QRadioButton" name="rHashNo">
                 <property name="text">
                  <string>None</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rHashMd5">
                 <property name="text">
                  <string>MD5</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rHashSha2">
                 <property name="text">
                  <string>SHA256</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="cModTimes">
                  <property name="text">
                   <string>Compare modification times</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="cDontHash">
                  <property name="text">
                   <string>Don't check hashes </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="cFixtime">
                  <property name="text">
                   <string>Fix file times instead of copy when the files are same according to hashes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="pExclFile">
            <property name="text">
             <string>Add exclude file...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pExclDir">
            <property name="text">
             <string>Add exclude directory...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pExclPath">
            <property name="text">
             <string>Add exclude path...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pDelEx">
            <property name="text">
             <string>Delete all excludes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelCmdLine">
       <property name="text">
        <string>unisync</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="workFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAbout">
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="unisync.qrc"/>
 </resources>
 <connections/>
</ui>
